JavaAssignment/src/arpit/AllDifferIndexOfArraysQ9.java

```diff
@@ -0,0 +1,80 @@
+package arpit;
+
+import java.util.Scanner;
+
+public class AllDifferIndexOfArraysQ9 {
+
+	// This function perform the operation to check array element differ first
+	// time at which index.
+	void arraysDifferAtFirstIndex(int[] element1, int[] element2) {
+
+		if (element1.length != element2.length) {
+			System.out.println("Araay length is not same");
+		} else {
+			for (int index = 0; index < element1.length; index++) {
+				if (element1[index] != element2[index]) {
+
+					// Display the result
+					System.out.println("value of Arrays is not matching at index is: " + "index " + index);
+
+				}
+			}
+		}
+
+	}
+
+	// This function return integral type Array
+	int[] inputForArray1() {
```

Comment:
Try to not write scanner class for each method. take input from the user in main method only.